lib/amorail/entities/company.rb

```diff
@@ -1,48 +1,41 @@
 module Amorail
   class AmoCompany < Amorail::AmoEntity
-    set_amo_name "company"
-    set_amo_response_name "contacts"
-    
+    set_amo_name 'company'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.